src/Tizen.Applications.WatchfaceComplication/Tizen.Applications/EditablesContainer.cs

```diff
@@ -0,0 +1,348 @@
+        /// <param name="editableId">The editable id.</param>
+        /// <exception cref="ArgumentException">Thrown when the invalid parameter is passed.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to invalid operation.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void Add(Complication comp, int editableId)
+        {
+            if (comp == null)
+                ErrorFactory.ThrowException(ComplicationError.InvalidParam);
+            if (IsExist(editableId))
+                ErrorFactory.ThrowException(ComplicationError.ExistID);
+
+            IEditable e = comp;
+            e.EditableId = editableId;
+            _compList.Add(comp);
+        }
+
+        /// <summary>
+        /// Removes the editable from edit list.
+        /// </summary>
+        /// <param name="editableId">The editable id.</param>
+        /// <exception cref="ArgumentException">Thrown when the invalid parameter is passed.</exception>
+        /// <example>
+        /// <code>
+        /// if (myContainer.IsExist(_colorEditId)
+        ///     myContainer.Remove(_colorEditId);
+        /// </code>
+        /// </example>
+        /// <since_tizen> 6 </since_tizen>
+        public void Remove(int editableId)
+        {
+            foreach (DesignElement de in _deList)
+            {
+                if (((IEditable)de).EditableId == editableId)
+                {
+                    _deList.Remove(de);
+                    return;
+                }
+            }
+
+            foreach (Complication comp in _compList)
+            {
+                if (((IEditable)comp).EditableId == editableId)
+                {
+                    _compList.Remove(comp);
+                    return;
+                }
+            }
+
+            ErrorFactory.ThrowException(ComplicationError.InvalidParam, "invalid editable ID");
+        }
+
+        internal IEditable GetEditable(int editableId)
+        {
+            foreach (IEditable ed in _deList)
+            {
+                if (ed.EditableId == editableId)
+                    return ed;
+            }
+
+            foreach (IEditable ed in _compList)
+            {
+                if (ed.EditableId == editableId)
+                    return ed;
+            }
+
+            return null;
+        }
+
+        internal DesignElement GetDesignElement(int editableId)
+        {
+            foreach (DesignElement de in _deList)
+            {
+                IEditable ed = de;
+                if (ed.EditableId == editableId)
+                    return de;
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Checks the editable with editableId is already exists in edit list.
+        /// </summary>
+        /// <param name="editableId">The target editable Id.</param>
+        /// <returns>true if the editable is already exists in edit list, otherwise false</returns>
```

Comment:
Suggestion:
true if the editable already exists in edit list, otherwise false

Exist is  an intransitive verb so you can't make passive voice.